client/src/components/login.component.js

```diff
@@ -0,0 +1,39 @@
+import React, { Component } from "react";
+import {Row,Col} from 'react-bootstrap';
+
+export default class Login extends Component {
+    render() {
+        return (
+<Row className="landing">
+<Col> <img src ="img/loginimg.png" alt="image"/> </Col>
```

Comment:
Instead of an `img` directory in `public`, can we use `assets` to serve all our static assets?